examples/erc20/Compound/cBAT.xml

```diff
@@ -24,9 +24,19 @@
 <ts:origins>
     <ts:ethereum contract="cBAT"/>
 </ts:origins>
-
+<ts:selection id="enabled" filter="allowance>0">
+    <ts:name>
+        <ts:string xml:lang="en">contract already enabled</ts:string>
+    </ts:name>
+</ts:selection>
+<ts:selection id="notEnabled" filter="allowance=0">
```

Comment:
@James-Sangalli One of this should be `filter="!(allowance=0)"` but I guess you haven't write it that way because we haven't included it yet.

Reply:
Stupid me. It’s correct